Lib/idlelib/idle_test/test_sidebar.py

```diff
@@ -248,61 +249,61 @@ def get_width():
         self.assert_sidebar_n_lines(1)
         self.assertEqual(get_width(), 1)
 
-    def test_click_selection(self):
-        self.linenumber.show_sidebar()
-        self.text.insert('1.0', 'one\ntwo\nthree\nfour\n')
-        self.root.update()
-
-        # Click on the second line.
-        x, y = self.get_line_screen_position(2)
-        self.linenumber.sidebar_text.event_generate('<Button-1>', x=x, y=y)
-        self.linenumber.sidebar_text.update()
-        self.root.update()
-
-        self.assertEqual(self.get_selection(), ('2.0', '3.0'))
-
-    def simulate_drag(self, start_line, end_line):
-        start_x, start_y = self.get_line_screen_position(start_line)
-        end_x, end_y = self.get_line_screen_position(end_line)
-
-        self.linenumber.sidebar_text.event_generate('<Button-1>',
-                                                    x=start_x, y=start_y)
-        self.root.update()
-
-        def lerp(a, b, steps):
-            """linearly interpolate from a to b (inclusive) in equal steps"""
-            last_step = steps - 1
-            for i in range(steps):
-                yield ((last_step - i) / last_step) * a + (i / last_step) * b
-
-        for x, y in zip(
-                map(int, lerp(start_x, end_x, steps=11)),
-                map(int, lerp(start_y, end_y, steps=11)),
-        ):
-            self.linenumber.sidebar_text.event_generate('<B1-Motion>', x=x, y=y)
-            self.root.update()
-
-        self.linenumber.sidebar_text.event_generate('<ButtonRelease-1>',
-                                                    x=end_x, y=end_y)
-        self.root.update()
-
-    def test_drag_selection_down(self):
-        self.linenumber.show_sidebar()
-        self.text.insert('1.0', 'one\ntwo\nthree\nfour\nfive\n')
-        self.root.update()
-
-        # Drag from the second line to the fourth line.
-        self.simulate_drag(2, 4)
-        self.assertEqual(self.get_selection(), ('2.0', '5.0'))
-
-    def test_drag_selection_up(self):
-        self.linenumber.show_sidebar()
-        self.text.insert('1.0', 'one\ntwo\nthree\nfour\nfive\n')
-        self.root.update()
-
-        # Drag from the fourth line to the second line.
-        self.simulate_drag(4, 2)
-        self.assertEqual(self.get_selection(), ('2.0', '5.0'))
+    # def test_click_selection(self):
```

Comment:
If you want to merge with these disabled, a comment should be added here as to why, referring to this PR, and a comment added to the corresponding sections of sidebar.py that the tests are disabled and must be re-enabled before editing the untested areas.

Reply:
I've added a comment to the tests.

The line numbers scrolling behavior is still at least partially covered by the htest, so perhpas such a comment in the code itself is unwarranted?